Sources/FrontEnd/TypeChecking/TypeChecker.swift

```diff
@@ -3646,9 +3627,41 @@ public struct TypeChecker {
     return r
   }
 
+  /// Returns the declarations in `decls` whose names match `n`.
```

Comment:
```suggestion
  /// Returns the members of `decls` whose names match `n`.
```

But what does it mean to "match" a name?  Is this

```suggestion
  /// Returns the members of `decls` named `n`.
```

?  If not, where is the meaning of "match" defined?